src/main/scala/org/broadinstitute/dsde/firecloud/webservice/LibraryApiService.scala

```diff
@@ -23,7 +23,15 @@ trait LibraryApiService extends HttpService with FireCloudRequestBuilding
 
   val libraryServiceConstructor: UserInfo => LibraryService
 
+  val duosAutocompleteUrl = FireCloudConfig.Duos.baseUrl + "/autocomplete?types=disease&q=%s" // does this still work?
```

Comment:
Yes, this query format works:
* https://consent-ontology.dsde-dev.broadinstitute.org/autocomplete?types=disease&q=cancer
* https://consent-ontology.dsde-consent-prod.broadinstitute.org/autocomplete?types=disease&q=cancer